src/nicolai_munsterhjelm_ex/ex05/bounded_sim.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Nicolai Munsterhjelm'
+__email__ = 'nicmust@nmbu.no'
+
+
+import random
+import walker_sim
+
+
+class BoundedWalker(walker_sim.Walker):
+    def __init__(self, start, home, left_limit, right_limit):
+        """
+        Initialise the walker
+
+        Arguments
+        ---------
+        start : int
+            The walker's initial position
+        home : int
+            The walk ends when the walker reaches home
+        left_limit : int
+            The left boundary of walker movement
+        right_limit : int
+            The right boundary  of walker movement
+        """
+        self.start = start
+        self.home = home
+        self.left_limit = left_limit
+        self.right_limit = right_limit
+
+        super().__init__(start, home)
```

Comment:
1. Put this (line 32) before assigning the other attributes.
2. remove what's in lines 27 and 28, as this is done by the `super().__init__()`